### PR TITLE
In 146491663 cross report comm method lovs

### PR DESCRIPTION
**Pull request by NateOSI**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -240,6 +246,7 @@ GEM
     timecop (0.8.1)
     tzinfo (1.2.3)
       thread_safe (~> 0.1)
+    uber (0.1.0)
```
Comment (aaronOSI) [inline Gemfile.lock]: gem considered harmful, please implement lyft

File: spec/representers/referral_representer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+              'gender' => participant_one.gender,
+              'id' => participant_one.id,
+              'last_name' => participant_one.last_name,
+              'legacy_id' => nil,
+              'legacy_source_table' => nil,
+              'middle_name' => participant_one.middle_name,
+              'name_suffix' => participant_one.name_suffix,
+              'roles' => participant_one.roles,
+              'screening_id' => screening.id,
+              'ssn' => '123456666'
```
Comment (jgedeon-osi) [inline spec/representers/referral_representer_spec.rb]: Should this be 'participant_one.ssn'

Comment (NateOSI) [inline spec/representers/referral_representer_spec.rb]: We strip hyphens from SSN's when we sent them to TPT

File: spec/representers/cross_report_representer_spec.rb [hunk 3]
```diff
@@ -22,14 +22,14 @@
       end
     end
 
-    context 'with a complete cross report' do
+    context 'with a comlete cross report except communication method' do
```
Comment (ThomasOSI) [inline spec/representers/cross_report_representer_spec.rb]: comlete => complete

File: app/mappings/communication_method.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# CommunicationMethod is a mapping class which maps cross report communication methods
+# in intake to their respective description and category in legacy
+class CommunicationMethod
+  MAPPING_VALUES = {
+    'Suspected Child Abuse Report' => {
+      category_id: 'XRPT_MTC', description: 'Suspected Child Abuse Report            '
```
Comment (saraOSI) [inline app/mappings/communication_method.rb]: Maybe `category_id` could be a constant in this class instead of repeated?

**Pull request merged**